core/server/models/settings.js

```diff
@@ -219,14 +220,22 @@ Settings = ghostBookshelf.Model.extend({
         });
     },
 
-    populateDefaults: function populateDefaults(unfilteredOptions) {
+    populateDefaults: async function populateDefaults(unfilteredOptions) {
         const options = this.filterOptions(unfilteredOptions, 'populateDefaults');
         const self = this;
 
         if (!options.context) {
             options.context = internalContext.context;
         }
 
+        // fetch available columns to avoid populating columns not yet created by migrations
+        const columnInfo = await ghostBookshelf.knex.table('settings').columnInfo();
+        const columns = Object.keys(columnInfo);
+
+        // fetch other data that is used when inserting new settings
+        const date = ghostBookshelf.knex.raw('CURRENT_TIMESTAMP');
+        const owner = await ghostBookshelf.model('User').getOwnerUser();
```

Comment:
I assumed we're OK using our models here because we're still technically in "app land" compared to straight migrations.

Reply:
👍  I think so